h/presenters.py

```diff
@@ -244,20 +241,13 @@ def description(self):
 
         """
         def get_selection():
-            targets = self.annotation.get("target")
-            if not isinstance(targets, list):
-                return
+            targets = self.annotation.target_selectors
```

Comment:
I don't think this is right. Annotations in our current database schema have a single target, and `target_selectors` is an array of selectors. So this code now will always return `None` because `target` is a selector and `target.get("selector")` will always be `None`.